### PR TITLE
Add the missing columns to the customer configuration list

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request adds the missing columns to the client config list

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no
_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

https://docs.magento.com/user-guide/configuration/catalog/xml-sitemap.html
https://docs.magento.com/user-guide/configuration/catalog/catalog.html

### CODE REVIEW

File: src/configuration/catalog/catalog.md [hunk 1]
```diff
@@ -205,6 +205,8 @@ Adobe Commerce supports several versions of Elasticsearch.
 |Elasticsearch Server Port|Global|Specifies the number of the server port used by Elasticsearch. Default value: `9200`|
 |Elasticsearch Index Prefix|Global|Assigns a prefix to identify the Elasticsearch index. Default value: `magento2`|
 |Enable Elasticsearch HTTP Auth|Global|If enabled, uses HTTP authentication to prompt for a username and password before accessing Elasticsearch Server. Options: Yes / No|
+|Elasticsearch HTTP Username|Global|Specifies the username for Elasticsearch HTTP authentication.|
+|Elasticsearch HTTP Password|Global|Specifies the password for Elasticsearch HTTP authentication.|
```
Comment (leslietilling) [inline src/configuration/catalog/catalog.md:209]:
```suggestion
|Elasticsearch HTTP Username|Global|When _Enable Elasticsearch HTTP Auth_ is set to `Yes`, specifies the username for Elasticsearch HTTP authentication.|
|Elasticsearch HTTP Password|Global|When _Enable Elasticsearch HTTP Auth_ is set to `Yes`, specifies the password for Elasticsearch HTTP authentication.|
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edit for clarity

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@leslietilling the Pull Request is successfully imported.

**Pull request merged**